sw/airborne/modules/lidar/tfmini.c

```diff
@@ -35,12 +36,20 @@
 #include "pprzlink/messages.h"
 #include "modules/datalink/downlink.h"
 
+// Test
+#include "firmwares/rotorcraft/autopilot_static.h"
+
+
+#define LIDAR_MIN_RANGE 0.1
+#define LIDAR_MAX_RANGE 12.0
+
+#define LIDAR_OFFSET 0.20   // Horizontal distance from the IMU to the LiDAR (in meters)
+#define LIDAR_HEIGHT 0.24   // Height of the LiDAR above the ground (in meters)
```

Comment:
I'm fine with adding offset values, but they should be configurable and set to 0 by default. Then use the airframe file to adjust to your own setup.